src/__tests__/reducers/followUnfollow/followUnfollow.test.js

```diff
@@ -0,0 +1,151 @@
+import {
+  FOLLOWING_USER_STARTED, FOLLOWING_USER_SUCCESS, FOLLOWING_USER_FAIL, FETCHING_FOLLOWING_LIST, FOLLOWING_LIST_LOADED, FOLLOWING_LIST_LOADED_FAILED, FOLLOWERS_LOADED,
+} from "../../../actions/types";
+import followUnfollow from "../../../reducers/followunfollow/followUnfollow";
+
+describe("Follow Unfollow Reducer Tests", () => {
+  const initialState = {
+    followers: [],
+    isFollowingSomeone: false,
+    msg: null,
+    following: [],
+    isLoading: false,
+  };
+
+  it("should return the initial state", () => {
+    const state = followUnfollow(undefined, {});
+    expect(state).toEqual({
+      followers: [],
+      isFollowingSomeone: false,
+      msg: null,
+      following: [],
+      isLoading: false,
+    });
+  });
+
+  it("should handle the FOLLOWING_USER_STARTED action", () => {
+    const action = {
+      type: FOLLOWING_USER_STARTED,
+    };
+    const loadedState = {
+      followers: [],
+      isFollowingSomeone: true,
+      msg: null,
+      following: [],
+      isLoading: false,
+    };
+    expect(followUnfollow(initialState, action)).toEqual(loadedState);
+  });
+
+  it("should follow a user successfully", () => {
```

Comment:
can change this title to `should allow a user to successfully follow another user`. Also consider reviewing other `it` titles